scripts/ice-road-pipeline.py

```diff
@@ -2,18 +2,20 @@
 Takes input directory full of .laz files and filters+classifies them to DTM laz and DTM tif.
 
 Usage:
-    ice-pipeline.py <in_dir> [-d debug] [-a asp_dir]
+    ice-road-pipeline.py <in_dir> [-e user_dem] [-d debug] [-a asp_dir]
 
 Options:
-    -d debug      turns on debugging logging  [default: True]
-    -a asp_dir    Directory with ASP binary files [default: False]
+    -e user_dem      Path to user specifed DEM [default: False]
```

Comment:
Here is where you would remove [default: False] and it will just make this False if the user doesn't enter this flag